internal/infrastructure/kubernetes/service.go

```diff
@@ -33,9 +33,13 @@ func (i *Infra) expectedService(infra *ir.Infra) (*corev1.Service, error) {
 	for _, listener := range infra.Proxy.Listeners {
 		for _, port := range listener.Ports {
 			target := intstr.IntOrString{IntVal: port.ContainerPort}
+			protocol := corev1.ProtocolTCP
```

Comment:
can you address a comment that we need E2E in the future?

Reply:
This can be converted into an issue, and it's not blocking. I will approve it.